common/kotlinx-coroutines-core-common/test/WithTimeoutOrNullTest.kt

```diff
@@ -7,10 +7,12 @@
 
 package kotlinx.coroutines.experimental
 
+import kotlinx.coroutines.experimental.channels.*
 import kotlin.coroutines.experimental.*
 import kotlin.test.*
 
 class WithTimeoutOrNullTest : TestBase() {
+
```

Comment:
Let's follow Kotlin code style with respect to empty lines (they should used to separate declarations in class, but not between class declaration line and first member). See https://kotlinlang.org/docs/reference/coding-conventions.html The only possible exception is when multi-line class declaration is formatted with a trailing `... {`, then an empty line should be used as a separator. However, I suggest we consistently prefer the second option from the conventions document and always put `{` on a separate line as a separator in such cases.